docker/nginx/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM nginx:alpine
+FROM pagespeed/nginx-pagespeed
```

Comment:
ngx_pagespeed is a poor fit for a dev-time Docker image. The image also (at time of writing) has not been updated in 3 years.

Best to stick with plain ol' nginx.